sympy/printing/tests/test_str.py

```diff
@@ -87,6 +87,7 @@ def test_Add():
     assert str(x - z*y**2*z*w) == "-w*y**2*z**2 + x"
     assert str(x - 1*y*x*y) == "-x*y**2 + x"
     assert str(sin(x).series(x, 0, 15)) == "x - x**3/6 + x**5/120 - x**7/5040 + x**9/362880 - x**11/39916800 + x**13/6227020800 + O(x**15)"
+    assert str(Add(Add(-w, x, evaluate=False), Add(-y, z,  evaluate=False),  evaluate=False)) == "(-w + x) + (-y + z)"
```

Comment:
Please add the following test cases that also validate the fix:
```
assert str(Add(Add(-x, -y, evaluate=False), -z, evaluate=False)) == "-z + (-x - y)"
assert str(Add(Add(Add(-x, -y, evaluate=False), -z, evaluate=False), -t, evaluate=False)) == "-t + (-z + (-x - y))"
```

Reply:
Everything looks good now but please make the title of your PR more descriptive as suggested above. You could say something like "Fix unevaluated printing of sum involving negative quantities." Also, add a release note, e.g. 

```
#### Release Notes
* printing
   
  * Fix str of sum involving negative quantities.
```

Reply:
That's okay. Note that the title of the PR is the one at the top of the page where it currently says "updated str.py #24030". You want to replace that with the description you added on the first line of your first message in this PR.